packages/box/box.ts

```diff
@@ -18,6 +18,9 @@ import { sandboxOptions, unboxOptions } from "typings";
 const normalizeURL = (
```

Comment:
Perhaps this method should be changed to something like `normalizeTarget` to signify that it won't necessarily be just a url. Thoughts?